gems/awesome-bot/CVE-2019-15224.yml

```diff
@@ -0,0 +1,21 @@
+---
+gem: awesome-bot
+url: https://nvd.nist.gov/vuln/detail/CVE-2019-15224
+date: 2019-08-20
+title: Critical severity vulnerability that affects rest-client, cron_parser, coin_base,
+  blockchain_wallet, awesome-bot, doge-coin, capistrano-colors, bitcoin_vanity, lita_coin,
+  coming-soon, and omniauth_amazon
+description: |-
+  The rest-client gem 1.6.10 through 1.6.13 for Ruby, as distributed on RubyGems.org, included a code-execution backdoor inserted by a third party.
+  Users of an affected version should consider downgrading to the last non-affected version of 1.6.9, or upgrading to 1.7.x.
+  Additionally, a set of other minor gems have been partially or completely yanked and are included in this advisory.
+  These include cron_parser, coin_base, blockchain_wallet, awesome-bot, doge-coin, capistrano-colors, bitcoin_vanity, lita_coin, coming-soon, and omniauth_amazon."
+unaffected_versions:
+  - "< 1.18.0"
+  - "> 1.18.0"
+cve: 2019-15224
```

Comment:
Please move `cve` and `ghsa` IDs to the top below `gem`.

Reply:
done